dowhy/interpreters/confounder_distribution_intepreter.py

```diff
@@ -0,0 +1,58 @@
+import seaborn as sns
+import numpy as np
+import matplotlib.pyplot as plt
+
+from dowhy.interpreters.visual_interpreter import VisualInterpreter
+from dowhy.causal_estimators.propensity_score_weighting_estimator import PropensityScoreWeightingEstimator
+from dowhy.causal_estimator import CausalEstimate
+
+
+class ConfounderDistributionIntepreter(VisualInterpreter):
+    SUPPORTED_ESTIMATORS = [PropensityScoreWeightingEstimator, ]
+
+    def __init__(self, estimate, **kwargs):
```

Comment:
you can add the parameters `fig_size`, '`font_name`, `var_name` as named parameters in the __init__() function. So the signature can look like: 
`def __init__(self, estimate, fig_size, font_name, var_name, **kwargs)`

That way python will automatically populate these variables with the user-given parameters (and any extra parameters from the user will still go to kwargs). Plus a user can look at the documentation of this class to find out the accepted parameters.